From blp@nicira.com Thu Sep 10 23:25:10 2009
Received: from FTLPMAILMX02.citrite.net (10.9.154.224) by
	LONPMAILMX02.citrite.net (10.30.224.163) with Microsoft SMTP Server (TLS)
	id 8.1.393.1; Thu, 10 Sep 2009 23:25:10 +0100
Received: from SMTP02.CITRIX.COM (10.9.154.239) by FTLPMAILMX02.citrite.net
	(10.9.154.224) with Microsoft SMTP Server id 8.1.393.1; Thu, 10 Sep 2009
	18:25:08 -0400
Received: from mail.geoviz.com (HELO leadville.nicira.com)
	([209.172.104.10]) by SMTP02.CITRIX.COM with ESMTP; 10 Sep 2009 18:25:07
	-0400
Received: from hardrock.nicira.com ([172.16.0.20])	by leadville.nicira.com
	with esmtp (Exim 4.67)	(envelope-from <blp@nicira.com>)	id
	1Mls4b-0005v7-B9; Thu, 10 Sep 2009 15:25:01 -0700
Received: from blp by hardrock.nicira.com with local (Exim 4.67)
	(envelope-from <blp@hardrock.nicira.com>)	id 1Mls4f-0004UX-7j; Thu, 10 Sep
	2009 15:25:05 -0700
From: Ben Pfaff <blp@nicira.com>
To: "jpettit@nicira.com" <jpettit@nicira.com>, Ian Campbell <Ian.Campbell@eu.citrix.com>
CC: Ben Pfaff <blp@nicira.com>
Date: Thu, 10 Sep 2009 23:24:48 +0100
Subject: [PATCH 3/6] ovs-vsctl: Be a little more positive about the code.
Thread-Topic: [PATCH 3/6] ovs-vsctl: Be a little more positive about the
	code.
Thread-Index: AcoyZY9RFjOXo38fTK+wJ3NXsMewwA==
Message-ID: <1252621491-17203-3-git-send-email-blp@nicira.com>
References: <1252621491-17203-1-git-send-email-blp@nicira.com>
In-Reply-To: <1252621491-17203-1-git-send-email-blp@nicira.com>
Accept-Language: en-GB, en-US
Content-Language: en-US
X-MS-Exchange-Organization-AuthAs: Anonymous
X-MS-Exchange-Organization-AuthSource: FTLPMAILMX02.citrite.net
X-MS-Has-Attach: 
X-MS-TNEF-Correlator: 
x-sbrs: None
x-mesageid: 64739372
x-ironport-server: ftlpip02.citrite.net
x-remote-ip: 209.172.104.10
x-policy: $LOW_THROTTLE
x-ironport-anti-spam-filtered: true
x-ironport-anti-spam-result: AhkFAF8ZqUrRrGgK/2dsb2JhbACBU5UnyEiEGAU
x-ironport-av: E=Sophos;i="4.44,366,1249272000";   
	d="scan'208";a="64739372"
Content-Type: text/plain; charset="iso-8859-1"
MIME-Version: 1.0
X-Evolution-Source: imap://ianca@lonpmailmx01.citrite.net/
Content-Transfer-Encoding: 8bit

---
 utilities/ovs-vsctl.in |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/utilities/ovs-vsctl.in b/utilities/ovs-vsctl.in
index d51d922..5f412a9 100755
--- a/utilities/ovs-vsctl.in
+++ b/utilities/ovs-vsctl.in
@@ -37,9 +37,9 @@ class Error(Exception):
         Exception.__init__(self)
         self.msg = msg
 
-### XXX This code is a bunch of crap.  There should be a
-### VSwitchConfiguration object with logically named fields and
-### methods instead.
+# XXX Most of the functions below should be integrated into a
+# VSwitchConfiguration object with logically named fields and methods
+# instead of this mishmash of functionality.
 
 # Locks 'filename' for writing.
 def cfg_lock(filename):
-- 
1.6.3.3

